src/java/fr/paris/lutece/plugins/example/modules/elastic/elasticdata/ProjectDataSource.java

```diff
@@ -0,0 +1,21 @@
+package fr.paris.lutece.plugins.example.modules.elastic.elasticdata;
+
+import java.util.Collection;
+
+import fr.paris.lutece.plugins.elasticdata.business.AbstractDataSource;
+import fr.paris.lutece.plugins.elasticdata.business.DataObject;
+import fr.paris.lutece.plugins.elasticdata.business.DataSource;
+
+/**
+ * My DataSource
+ */
+public class ProjectDataSource extends AbstractDataSource implements DataSource
+{
+
+    @Override
+    public Collection<DataObject> getDataObjects( )
+    {
+        ElasticProjectDao dao = new ElasticProjectDao();
+        return dao.selectAll();
```

Comment:
The DAO should be a singleton injected by spring (I see the wiki needs to be updated as well)